index.html

```diff
@@ -212,6 +212,7 @@ <h2>I'm always interested in hearing about new projects,
                 <input id="Email" type="email" placeholder="Email address" name="user-email" required>
                 <label class="label" for="Message">Enter your text here:</label>
                 <textarea id="Message" name="user-message" placeholder="Enter text here" required maxlength="500"></textarea>
+                <small></small>
                 <button type="submit">Get in touch</button>
             </form>
             <form id="desktop-form" action="https://formspree.io/f/xknylodz" method="POST">
```

Comment:
- [ ] [OPTIONAL] Your validation form working very well :clap:  However I would like to recommend you arrange your contact form in order to make it similar to the Figma template and also give a great user experience :smile: 

### Your contact form
![Screenshot from 2022-07-20 22-27-13](https://user-images.githubusercontent.com/6615394/180093441-8fc9a8ce-6e26-4018-8c96-f465a980a932.png)

### Figma 
![Screenshot from 2022-07-20 22-25-28](https://user-images.githubusercontent.com/6615394/180093446-5d13e91d-96d1-49f9-a02a-5f495e97bcd7.png)